File: P/sortList.py
# -*- coding: utf-8 -*-
"""
Created on Wed Sep 23 20:29:05 2020

@author: Mami
"""

def sortList(L, n):
    L2 = []
    counter = 0
    while (counter <= n):
        m, idx = searchMinFromList(L, n)
        L2.append(m)
        del L[idx]
        n = n-1
        counter = counter + 1
    return L2

File: M/sortList.m
function [L2] = sortList(L)
% searchMinFromList
% Input Arguments
% L
% Output Arguments
% L2

n = length(L);
L2 = zeros(1,n);
counter = 1;
while counter <= n
    [m, idx] = searchMinFromList(L);
    L2(counter) = m;
    L(idx) = [];
    counter = counter + 1;
end

end

